defaults/main.yml

```diff
@@ -37,12 +37,19 @@ elasticsearch_yum_repo_baseurl: "https://artifacts.elastic.co/packages/{{ elasti
 
 ###########
 ##  Plugins
-elasticsearch_update_plugins:  false
-elasticsearch_plugins:         []
+elasticsearch_update_plugins:  true
```

Comment:
Will this setting mean that any existing plugins will be automatically updated to their latest version when this role is run? This would mean we would have to check/test any new plugin versions before running the `infra-tools` build.

Reply:
I agree. Leaving it for false by default.